ganttproject/src/main/java/net/sourceforge/ganttproject/storage/SqlProjectDatabaseImpl.kt

```diff
@@ -97,9 +98,11 @@ class SqlProjectDatabaseImpl(private val dataSource: DataSource) : ProjectDataba
 
   /** Add a query to the current txn. Executes immediately if no transaction started. */
   private fun withLog(buildQuery: (dsl: DSLContext) -> String,
-                      buildUndoQuery: (dsl: DSLContext) -> String) {
-    val query = SqlQuery(buildQuery(DSL.using(SQLDialect.H2)), buildQuery(DSL.using(SQLDialect.POSTGRES)))
-    val undoQuery = SqlQuery(buildUndoQuery(DSL.using(SQLDialect.H2)), buildUndoQuery(DSL.using(SQLDialect.POSTGRES)))
+                      buildUndoQuery: (dsl: DSLContext) -> String,
+                      postgresOperationDto: OperationDto,
```

Comment:
this is not really "postgres" dto, is it? It is rather DB-independent.

Reply:
Yup, let's rename them to smth like `colloboqueOperationDto` and `colloboqueUndoOperationDto`